DotNet-Platform.md

```diff
@@ -0,0 +1,40 @@
+![Logo][logo]
+
+# The .NET Platform 
+
+**.NET platform** is big and comprehensive collection of frameworks and libraries manager by [Microsoft](https://www.microsoft.com/net). In the beginning it could be quite daunting to get familiar with .Net ecosystem.
+
+Our aim is to get you familiar with the .Net platform, so that you are ready to learn the [c# programing language](C-Sharp) along with development environment, project structures, techniques and best practices you will be performing as a professional developer.
+
+Let's get some of the jargon's out of the way, don't worry, you don't need to know in-depth about these at this point.
+
+* [.Net Framework](https://msdn.microsoft.com/en-gb/library/w0x726c2(v=vs.110).aspx) : (_pronounced dot net_) provides configuration for your application to run on top of it. 
+It provides all the services an app needs for instance communication with database, networks, file systems which are commonly used to build console, desktop, web, mobile and game applications.
+
+ * CLR (Common Language Runtime) : is an execution environment for your application, which manages it's life-cycle.
```

Comment:
Link to an external Microsoft resource which officially defines or explains what CLR is.